iocore/utils/I_Machine.h

```diff
@@ -31,6 +31,8 @@
 #ifndef _I_Machine_h
 #define _I_Machine_h
 
+#include <unordered_set>
```

Comment:
I suspect, but lets check with @SolidWallOfCode , that we should not use STL hashes here. We have internal implementations that we prefer. However, this might not be performance sensitive ?